disperser/apiserver/server_v2.go

```diff
@@ -277,8 +277,109 @@ func (s *DispersalServerV2) RefreshOnchainState(ctx context.Context) error {
 }
 
 // GetPaymentState returns the payment state for a given account and the related on-chain parameters
-// Deprecated: Use GetPaymentStateForAllQuorums instead.
+// Deprecating soon: use GetPaymentStateForAllQuorums instead.
```

Comment:
Keep the `Deprecated: ` syntax because IDEs pick up on that. You can add additional context after it. See https://go.dev/wiki/Deprecated